fleetspeak/src/server/mysql/clientstore.go

```diff
@@ -361,7 +361,18 @@ func (d *Datastore) RecordResourceUsageData(ctx context.Context, id common.Clien
 	})
 }
 
-func (d *Datastore) FetchResourceUsageRecords(ctx context.Context, id common.ClientID, limit int) ([]*spb.ClientResourceUsageRecord, error) {
+func (d *Datastore) FetchResourceUsageRecords(ctx context.Context, id common.ClientID, startTimestamp, endTimestamp *tspb.Timestamp) ([]*spb.ClientResourceUsageRecord, error) {
+	startTimeRange, parseErr := ptypes.Timestamp(startTimestamp)
```

Comment:
In Go, it's typical to use "err" for all kinds of errors. No need to introduce "parseErr", if "err" can be used.